src/main/java/com/glencoesoftware/omero/ms/image/region/ImageRegionMicroserviceVerticle.java

```diff
@@ -353,6 +360,9 @@ public void deploy(JsonObject config, Promise<Void> prom) {
         router.get(
                 "/webclient/annotation/:annotationId*")
             .handler(this::downloadFileAnnotation);
+        router.head(
+                "/webclient/annotation/:annotationId*")
+            .handler(this::fileAnnotationMetadata);
```

Comment:
A `HEAD` request should return the exact headers that a `GET` request would return, so I wonder if the same method should handle these requests and just not return the response body if the method was `HEAD`, to avoid code duplication?

Reply:
I can try to get them closer, but obviously `content-range` is not going to make sense for a `HEAD` request. I'll do my best to consolidate though

Reply:
I do think it makes sense and can be useful to have `content-range` in the response to a `HEAD` request, to see what byte range would be returned. It's quite possible that we would not use this functionality, but PathViewer will be using third-party libraries to do these requests, so who knows. We can wait with implementing more features until we know if we need them.